workspaces/sonarqube/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "private": true,
   "engines": {
-    "node": "18 || 20"
+    "node": "18 || 20 || 22"
```

Comment:
Can we revert this change? The plan is to do this as well as update the CI in the new year.